tests/e2e/utils/pages/setup-mc/step-1-set-up-accounts.js

```diff
@@ -428,4 +447,35 @@ export default class SetUpAccountsPage extends MockRequests {
 	getAdsAccountConnectedText() {
 		return this.getGoogleAdsAccountCard().getByText( 'Connected' );
 	}
+
+	/**
+	 * Get Ads account connected notice text.
+	 *
+	 * @return {import('@playwright/test').Locator} Get ads account connected text.
+	 */
+	getAdsAccountConnectedNotice() {
+		return this.getGoogleAdsAccountCard().getByText(
+			'Conversion measurement has been set up. You can create a campaign later.'
+		);
+	}
+
+	/**
+	 * Get ads claim account button.
+	 *
+	 * @return {import('@playwright/test').Locator} Get ads account Claim Account button.
+	 */
+	getAdsClaimAccountButton() {
+		return this.getClaimAdsAccountButton();
```

Comment:
This method is duplicated in the following.
https://github.com/woocommerce/google-listings-and-ads/blob/bb3a22209f363ade918413e39b2344e42b83e0ec/tests/e2e/utils/pages/setup-mc/step-1-set-up-accounts.js#L417-L424

What about leaving this one and removing another one?

```suggestion
		return this.page.getByRole( 'button', {
			name: 'Claim Account',
			exact: true,
		} );
```